ethexe/db/src/database.rs

```diff
@@ -43,26 +43,24 @@ use gprimitives::H256;
 use parity_scale_codec::{Decode, Encode};
 use std::collections::{BTreeMap, BTreeSet, VecDeque};
 
-const LOG_TARGET: &str = "ethexe-db";
-
 #[repr(u64)]
 enum KeyPrefix {
```

Comment:
what if we somehow highlight usage of these prefixes: on types (preferrable) or at least with comments which keys must be used with/without router address and some additional keys ?

Reply:
Sorry, but I cannot understand what do you mean here

Reply:
Waiting

Reply:
Discussed